OptimizelySDK/Utils/Validator.cs

```diff
@@ -43,6 +45,13 @@ public static bool ValidateJSONSchema(string configJson, string schemaJson = nul
                 return false;
             }
         }
+#endif
+#if NET35
+        public static bool ValidateJSONSchema(string configJson, string schemaJson = null)
+        {
+            return true;
```

Comment:
I think we should either remove the function altogether, or throw a NotImplemented Exception.  Returning either true or false when not doing any validation is wrong.